src/query/sql/src/planner/binder/table.rs

```diff
@@ -329,7 +330,8 @@ impl Binder {
                     .get(QUERY)
                     .ok_or_else(|| ErrorCode::Internal("Invalid VIEW object"))?;
                 let tokens = tokenize_sql(query.as_str())?;
-                let (stmt, _) = parse_sql(&tokens, self.dialect)?;
```

Comment:
I'm not sure it's a good way to fix the flaky test. if sql_dialect will cause the test err, better to add `set sql_dialect='PostgreSQL'` in the begin of test?